README.md

```diff
@@ -9,3 +9,12 @@ The GraalVM Reachability Metadata Repository enables Native Image users to share
 
 We welcome contributions from the community. Before submitting a pull
 request, please [review our contribution guide](./CONTRIBUTING.md).
+
+
+### Tested Libraries and Frameworks
+
+We track the libraries and frameworks that are tested with GraalVM Native Image. If you would like to see your library/framework
+in the list too, please contribute (open a pull request) to the [GraalVM Reachability Metadata Repository](https://github.com/oracle/graalvm-reachability-metadata).
+Before submitting a pull request, please read [this guide](./CONTRIBUTING.md#tested-libraries-and-frameworks).
+
+Every library/framework that is included in the list, will receive an appropriate badge with a test level indicator, as a confirmation that the library/framework is tested with Native Image.
```

Comment:
We don't have a badge yet, so this needs to be removed for now. Maybe we should just duplicate here what we have in the `Contributing.md`:
```
## Tested Libraries and Frameworks

If you would like to add your project to the list of libraries or frameworks tested with GraalVM Native Image, please open a pull request and add an entry to [this file](https://github.com/oracle/graalvm-reachability-metadata/blob/master/library-and-framework-list.json) according to [this schema](https://github.com/oracle/graalvm-reachability-metadata/blob/master/library-and-framework-list-schema.json). Entries are listed on the [GraalVM website](https://graalvm.org/native-image/libraries-and-frameworks/) and may be used for tools in the future.
```